bots/image-create

```diff
@@ -57,17 +56,18 @@ if args.image in ["candlepin", "continuous-atomic", "fedora-atomic", "ipa", "rhe
 
 class MachineBuilder:
     def __init__(self, machine):
+        tempdir = testvm.get_temp_dir()
         self.machine = machine
 
         try:
-            os.makedirs(TEMP, 0o750)
+            os.makedirs(tempdir, 0o750)
```

Comment:
Valid, but could also use the `exist_ok=True` option now.

Reply:
Sure.